cmd/sealos/cmd/run.go

```diff
@@ -92,7 +84,6 @@ func newRunCmd() *cobra.Command {
 		},
 	}
 	runArgs.RegisterFlags(runCmd.Flags())
-	runCmd.Flags().BoolVar(&runSingle, "single", false, "run cluster in single mode")
```

Comment:
Parameter deletion is an important change, which will cause old users to be at a loss, so it should be reserved and remind users that this parameter will be deleted in the future, and the default is single